content/pt-br/about.md

```diff
@@ -15,11 +15,11 @@ Nós fornecemos gratuitamente os certificados digitais necessários para habilit
 
 Os princípios chave do Let's Encrypt são: 
```

Comment:
"da Let's Encrypt" :-)

Reply:
Weird how my text search engine didn't spot this, gonna fix it shortly.